hibernate-core/src/test/java/org/hibernate/test/locking/PessimisticWriteLockTimeoutTest.java

```diff
@@ -54,6 +56,7 @@ public void createTestData() {
 	@Test
 	@RequiresDialect({ Oracle8iDialect.class, PostgreSQL81Dialect.class,
 			SQLServer2005Dialect.class } )
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
I think `@RequiresDialectFeature` can be removed

Reply:
done